app/src/main/java/com/udacity/bakingapp/activity/StepActivity.java

```diff
@@ -20,13 +20,17 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         final Bundle arguments = getIntent().getExtras();
         Timber.d("Received bundle in StepActivity: " + arguments);
 
-
-        final RecipeStepFragment recipeStepFragment = new RecipeStepFragment();
-        recipeStepFragment.setArguments(arguments);
-        getSupportFragmentManager()
-                .beginTransaction()
-                .add(R.id.recipe_step_placeholder, recipeStepFragment)
-                .commit();
+        // Only create fragment if does not exist
```

Comment:
nit: Comment is inconsistent with what's used in RecipeActivity for same fragment